openerp/models.py

```diff
@@ -2434,7 +2434,7 @@ def _auto_init(self, cr, context=None):
 
         store_compute = False
         stored_fields = []              # new-style stored fields with compute
-        todo_end = []
+        todo_end = context['todo']
```

Comment:
A history of method overrides ignoring (and not forwarding) the value returned by the super call, I think.

If the return value is unsuitable, I'd rather the method took a todos parameter though.